flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -1003,8 +1026,9 @@ class FMCMainDisplay extends BaseAirliners {
                 case FmgcFlightPhases.CRUISE: {
                     let speed = this.managedSpeedCruise;
 
-                    if (SimVar.GetSimVarValue("INDICATED ALTITUDE", "feet") < this.managedSpeedLimitAlt) {
-                        speed = Math.min(speed, this.managedSpeedLimit);
+                    // TODO should this actually be applied in cruise phase?
+                    if (this.climbSpeedLimit !== undefined && SimVar.GetSimVarValue("INDICATED ALTITUDE", "feet") < this.climbSpeedLimitAlt) {
```

Comment:
iirc this is the global 250kt below 10.000ft limit (which iirc can be modified)

Reply:
Yes, this PR implements the ability to modify it, and also splits the climb and des speed limits (hence the uncertainty about which applies in cruise...).